File: CS50_2022/pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    // Get the user to input a number between 1 and 8 inclusive
    do
    {
        n = get_int("Height: ");
    }
    while ((n < 1) || (n > 8));
    for (int i = 1; i <= n; i++)
    {
        // Outputs spaces
        for (int k = 0; k < n - i; k++)
        {
            printf(" ");
        }
        // Outputs hashes
        for (int j = 0; j < (n - (n - i)); j++)
        {
            printf("#");
        }
        printf("  ");
	for (int j = 0; j < (n - (n - i)); j++)
	{
	    printf("#");
	}
	printf("\n");
    }
}

File: CS50_2022/pset6/sentimental-mario-less/mario.py
from cs50 import get_int


# Gets the height from the user
# and ensures the input is between 0 and 9 exclusively
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(1, height+1, 1):
    # Prints out the spaces
    for j in range(0, height - i, 1):
        print(" ", end="")
    # Prints out the hashes
    for k in range(0, i, 1):
        print("#", end="")
    print()
